src/ui-lib/sass/06-utilities/_has-centered-text.scss

```diff
@@ -0,0 +1,3 @@
+.grav-u-has-centered-text {
```

Comment:
This is an interesting name for a css class? Isn't this naming convention usually used for method that return a boolean?